pkg/resources/password_policy_acceptance_test.go

```diff
@@ -1,49 +1,64 @@
 package resources_test
 
 import (
-	"fmt"
 	"strings"
 	"testing"
 
 	acc "github.com/Snowflake-Labs/terraform-provider-snowflake/pkg/acceptance"
+	"github.com/hashicorp/terraform-plugin-testing/config"
 	"github.com/hashicorp/terraform-plugin-testing/helper/acctest"
 	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+	"github.com/hashicorp/terraform-plugin-testing/tfversion"
 )
 
 func TestAcc_PasswordPolicy(t *testing.T) {
 	accName := strings.ToUpper(acctest.RandStringFromCharSet(10, acctest.CharSetAlpha))
+	m := func(minLength int, maxLength int, comment string) map[string]config.Variable {
+		return map[string]config.Variable{
+			"name":       config.StringVariable(accName),
+			"database":   config.StringVariable(acc.TestDatabaseName),
+			"schema":     config.StringVariable(acc.TestSchemaName),
+			"min_length": config.IntegerVariable(minLength),
+			"max_length": config.IntegerVariable(maxLength),
+			"comment":    config.StringVariable(comment),
```

Comment:
to make it more complete maybe add new fields?

Reply:
there are not all fields specified in the test (even not all previous fields), so I think it's fine to test just a few